chart/README.md

```diff
@@ -205,3 +205,9 @@ Specify each parameter using the `--set key=value[,key=value]` argument to `helm
 | `tls`                          | "ingress"                                             | ***string*** - See External TLS Termination for details. - *"ingress, external"*                                                                                                                             |
 | `systemDefaultRegistry`        | ""                                                    | ***string*** - private registry to be used for all system Docker images, e.g., [http://registry.example.com/] *Available as of v2.3.0*                                                                       |
 | `useBundledSystemChart`        | false                                                 | ***bool*** - select to use the system-charts packaged with Rancher server. This option is used for air gapped installations.  *Available as of v2.3.0*                                                       |
+| `customLogos.enabled`          | false                                                 | ***bool*** - Enabled [Rancher custom logos](https://github.com/rancher/ui/tree/master/public/assets/images/logos) pvc. K8s storageClass (dynamic pv) or existingClaim (static pv) are required|
```

Comment:
>  K8s storageClass (dynamic pv) or existingClaim (static pv) are required|

Thats only for PVC right ? We should say: `If using a PVC then K8s...`

Reply:
It's for enabling persistence, pvc or configmap. Updated